docspec/src/docspec/__init__.py

```diff
@@ -286,6 +345,23 @@ class Class(HasMembers):
   Represents a class definition.
   """
 
+  class Semantic(enum.Enum):
```

Comment:
Sorry, I'm giving my 2 cents. 

Why did you add this to the `HasMember` object and not the `Class` object? I don't see a module having all those properties.

Reply:
@tristanlatr I think you are mistaken, the line you are highlighting here is inside the definition of `Class`. 🙂

Reply:
I can see how, though. The diff is not very friendly in that regard.

![image](https://user-images.githubusercontent.com/1318438/131391883-7e0b6561-d9e7-455a-8333-3ddcbf810cef.png)

![image](https://user-images.githubusercontent.com/1318438/131391918-968d0e5e-5c22-4d9b-ae1d-0063f64f3898.png)

Reply:
My bad!